manage.py

```diff
@@ -138,10 +138,12 @@ def prepare_server(self) -> None:
 
         print("Applying migrations.")
         call_command("migrate", verbosity=self.verbosity)
-        print("Collecting static files.")
-        call_command("collectstatic", interactive=False, clear=True, verbosity=self.verbosity)
 
         if self.debug:
+            # In Production, collectstatic is ran in the Docker image
+            print("Collecting static files.")
+            call_command("collectstatic", interactive=False, clear=True, verbosity=self.verbosity)
```

Comment:
Right now, due to the verbosity of this command, reloading the server generates about 200 log lines, saying the existing assets have been deleted. I think it is causing more harm than good, so I propose to reduce the verbosity by one.

```suggestion
            call_command("collectstatic", interactive=False, clear=True, verbosity=self.verbosity - 1)
```

With this suggestion applied, it only generates one log line, `Collecting static files.`.